src/app/client/src/app/modules/explore-page/components/explore-page/explore-page.component.html

```diff
@@ -3,9 +3,19 @@
 <div [ngClass]="layoutConfiguration ? 'sbt-fluid-content-bg':''">
   <div class="sb-g" [ngClass]="layoutConfiguration ? 'sb-g sbt-container sbt-page-content' : 'sb-g'"
     [appTelemetryImpression]="telemetryImpression">
-    <div [ngClass]="FIRST_PANEL_LAYOUT">
+    <!-- Filter for web version -->
+    <!-- `isFilterEnabled` reference from current page data -->
+    <div *ngIf="!isDesktopApp && isFilterEnabled" [ngClass]="FIRST_PANEL_LAYOUT">
+      <div [ngClass]="isDesktopApp ? 'sb-desktop-filter-section' : ''">
+        <app-search-filter [facets$]="facets$" [layoutConfiguration]="layoutConfiguration" [isOpen]='true' *ngIf="initFilter"
+        (filterChange)="getFilters($event)" [pageData]="getCurrentPageData()" [pageId]="isUserLoggedIn() ? 'resource-page' : 'explore-page'" [defaultFilters]="defaultFilters">
+      </app-search-filter>
+    </div>
+  </div>
+  <!-- Filter for desktop version -->
+    <div *ngIf="isDesktopApp" [ngClass]="FIRST_PANEL_LAYOUT">
         <div *ngIf="layoutConfiguration">
-          <div *ngIf="isDesktopApp">
+          <div>
```

Comment:
please do not remove the *ngIf as this is needed for desktop app

Reply:
No; it is moved to line number 17.